src/Views/Form/Templates/Sequoia/optionConfig.php

```diff
@@ -104,6 +104,17 @@
 				],
 				'default'    => __( 'Continue', 'give' ),
 			],
+			[
+				'id'      => 'decimals_enabled',
+				'name'    => __( 'Decimal amounts', 'give' ),
+				'desc'    => __( 'Do you want to enable decimal amounts?', 'give' ),
```

Comment:
I think we can make the description text more helpful. In this case, when the setting is disabled decimal values are rounded. That is probably helpful information.

Reply:
Resolved: 1647d254